lib/quantum/job_broadcaster.ex

```diff
@@ -252,7 +326,19 @@ defmodule Quantum.JobBroadcaster do
         "[#{inspect(Node.self())}][#{__MODULE__}] Deleting all jobs"
       end)
 
-    messages = for {name, %Job{state: :active}} <- jobs, do: {:remove, name}
+    messages =
+      for {name, %Job{state: :active} = job} <- jobs do
+        # Send event to telemetry incase the end user wants to monitor events
+        :telemetry.execute([:quantum, :job, :delete], %{}, %{
```

Comment:
This will only be executed for active jobs. I think we also want to call it on inactive ones.